core/math/color_names.inc

```diff
@@ -24,7 +24,7 @@ static NamedColor named_colors[] = {
 	{ "BLANCHED_ALMOND", Color(1.00, 0.92, 0.80) },
 	{ "BLUE", Color(0.00, 0.00, 1.00) },
 	{ "BLUE_VIOLET", Color(0.54, 0.17, 0.89) },
-	{ "BROWN", Color(0.65, 0.16, 0.16) },
+	{ "BROWN", Color(0.647, 0.164, 0.164) },
```

Comment:
Maybe it's better to define these colors with RGBA hex values from the standard, like:

```suggestion
	{ "BROWN", Color::hex(0xA52A2AFF) },
```

Reply:
I can transform them all if that's what we want. We can also use the Color("a52a2a") format as well.
Also X11 supports both `DIM GRAY` and `DIM GREY` (same color, two names) and we only support `dim gray`, do we want the other one?